boards/nucleo-wl55jc/doc.txt

```diff
@@ -0,0 +1,51 @@
+/**
+@defgroup    boards_nucleo-wl55jc STM32 Nucleo-wl55jc
```

Comment:
```suggestion
@defgroup    boards_nucleo-wl55jc STM32 Nucleo-WL55JC
```

To be in line with other nucleo `BOARD`s https://9245-7370241-gh.circle-artifacts.com/0/doc/group__boards__common__nucleo64.html